WordPress/Sniffs/PHP/IniSetSniff.php

```diff
@@ -0,0 +1,173 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+namespace WordPressCS\WordPress\Sniffs\PHP;
+
+use WordPressCS\WordPress\AbstractFunctionParameterSniff;
+use \PHP_CodeSniffer\Util\Tokens;
```

Comment:
One more... just noticed you're not actually using the `Tokens` class in this sniff, so this `use` statement should be removed.